.github/workflows/reusable-unit.yml

```
@@ -77,11 +78,11 @@ jobs:
         run: yarn ui build
 
       - name: Build react-core package
-        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' || matrix.package == 'react-storage' || matrix.package == 'react-liveness' }}
+        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' || matrix.package == 'react-geo' || matrix.package == 'react-storage' || matrix.package == 'react-liveness' }}
```

Since this is getting pretty long, let's use multiline `if`:
```suggestion
        if: |
          matrix.package == 'react' || 
          matrix.package == 'react-native' || 
          matrix.package == 'react-geo' || 
          matrix.package == 'react-storage' || 
          matrix.package == 'react-liveness' 
```

Note: you may omit `${{ ... }}` in `if` statement ([docs](https://docs.github.com/en/actions/learn-github-actions/expressions)). Same with line 85 below!